### PR TITLE
[stable/redis-ha]: added ability to specify exporter lua script

### PR DESCRIPTION
**Pull request by jwenz723**
#### Is this a new chart
No

#### What this PR does / why we need it:
Added the ability to specify a lua script that will be utilized by prometheus-redis-exporter. This functionality is mirroring what is in the [stable/prometheus-redis-exporter](https://github.com/helm/charts/tree/master/stable/prometheus-redis-exporter) helm chart: https://github.com/helm/charts/blob/b126482a391078683ebd0a0878552b0045bd1360/stable/prometheus-redis-exporter/values.yaml#L57-L60

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by jwenz723** [comment]
Adding maintainers: @ssalaues @DandyDeveloper

File: stable/redis-ha/README.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ The following table lists the configurable parameters of the Redis chart and the
...
 | `exporter.annotations`    | Prometheus scrape annotations                                                                                                                                                                            | `{prometheus.io/path: /metrics, prometheus.io/port: "9121", prometheus.io/scrape: "true"}` |
 | `exporter.extraArgs`      | Additional args for the exporter                                                                                                                                                                         | `{}`                                                                                       |
+| `exporter.script.configmap`             | Let you run a custom lua script from a configmap. The corresponding environment variable `REDIS_EXPORTER_SCRIPT` will be set automatically                                                 |                                                                                            |
```
Comment (DandyDeveloper) [inline stable/redis-ha/README.md]: This is too verbose (although helpful): 

```suggestion
| `exporter.script.configmap`             | Run custom Lua script as ConfigMap. Sets env var `REDIS_EXPORTER_SCRIPT`.                                                 |                                                                                            |
```

File: stable/redis-ha/README.md [hunk 2]
```diff
@@ -93,6 +93,8 @@ The following table lists the configurable parameters of the Redis chart and the
...
+| `exporter.script.keyname`               | Name of the key inside configmap which contains your script                                                                                                                                |                                                                                            |
```
Comment (DandyDeveloper) [inline stable/redis-ha/README.md]: I wonder if this is too flexible, may be better just to have a static key as you can't reference multiple scripts.

File: stable/redis-ha/values.yaml [hunk 3]
```diff
@@ -259,6 +259,11 @@ exporter:
   # Additional args for redis exporter
   extraArgs: {}
 
+  # Used to mount a LUA-Script via config map and use it for metrics-collection
+  # script:
+  #   configmap: prometheus-redis-exporter-script
```
Comment (DandyDeveloper) [inline stable/redis-ha/values.yaml]: I think if we're going to do this, we need to have the chart create the ConfigMap and write the script as the value to the configmap so things can be set more statically. This make things easy for people to make mistakes. 

People that are uninformed may not understand they have to define their own ConfigMap template. 

I'm open to having my mind changed here.

**Review by DandyDeveloper — CHANGES_REQUESTED** [review]
Great idea to include the script, but if we want to make this a change in the chart. We should support most of the template rendering. See review comments.

File: stable/redis-ha/README.md (hunk 1 above)
Comment (jwenz723) [reply stable/redis-ha/README.md]: Thanks for the suggestion, I had just copied the description exactly from the prometheus-redis-exporter chart README.

File: stable/redis-ha/values.yaml (hunk 3 above)
Comment (jwenz723) [reply stable/redis-ha/values.yaml]: I like this idea, I had just copied over exactly what was being done in the prometheus-redis-exporter chart. I'll push a change with your suggestion, seems easier to use.

Comment (jwenz723) [reply stable/redis-ha/values.yaml]: @DandyDeveloper I have implemented your suggestion, let me know what you think.

**Comment by DandyDeveloper** [comment]
@jwenz723 Looks great. Thanks and sorry for the delay.

**Pull request merged**